test/widgets/content_test.dart

```diff
@@ -27,6 +27,21 @@ import 'dialog_checks.dart';
 import 'message_list_checks.dart';
 import 'page_checks.dart';
 
+/// Set [debugNetworkImageHttpClientProvider] to return a constant image.
+///
+/// Returns the [FakeImageHttpClient] that handles the requests.
+///
+/// The caller must set [debugNetworkImageHttpClientProvider] back to null
+/// before the end of the test.
+FakeImageHttpClient prepareBoringImageHttpClient() {
```

Comment:
nit: prefix for this commit:

f3aebc9fd autocomplete [nfc]: Pull `prepareBoringImageHttpClient` out to toplevel

should be `autocomplete test [nfc]:`, to indicate that it's only touching the test code

(And then the "nfc" part means that on top of that, it has only an NFC effect on even the test code.)

Reply:
Oh and looking again, it's not really about the autocomplete tests in particular, so `autocomplete` isn't quite right.

I'll make it `content test [nfc]`, since it is in that file. Could also have made it just `test [nfc]`, since the point is it's now exporting this for other tests.